From 89deffd16b231ead92e437d9b58e1149ac0c1e10 Mon Sep 17 00:00:00 2001
From: Badri Narayanan <mbadnara@qti.qualcomm.com>
Date: Tue, 21 Jul 2026 11:35:39 -0700
Subject: [PATCH 3/4] Allow users to specify arm64ReproDir by cmake flag

---
 cmake/arm64x.cmake | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/cmake/arm64x.cmake b/cmake/arm64x.cmake
index 525206367..9cd26e9bc 100644
--- a/cmake/arm64x.cmake
+++ b/cmake/arm64x.cmake
@@ -1,4 +1,5 @@
-set(arm64ReproDir "${CMAKE_CURRENT_SOURCE_DIR}/repros")
+set(arm64ReproDir "${CMAKE_CURRENT_SOURCE_DIR}/repros" CACHE PATH "Path to the ARM64 repro directory for ARM64X build")
+message(STATUS "Using arm64ReproDir: " ${arm64ReproDir})
 
 function(set_arm64_dependencies n)
 	set(ARM64_LIBS)
-- 
2.53.0.windows.1

