From c60a332e9a65dbae2920914adb8537de60f566e7 Mon Sep 17 00:00:00 2001
From: "Cheng-Hsin Weng (Brian)" <chenweng@qti.qualcomm.com>
Date: Fri, 13 Mar 2026 15:57:10 +0800
Subject: [PATCH 4/4] Update argument for monolithic lstm

---
 onnxruntime/test/onnx/main.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/onnxruntime/test/onnx/main.cc b/onnxruntime/test/onnx/main.cc
index f08b974d60..a50d771234 100644
--- a/onnxruntime/test/onnx/main.cc
+++ b/onnxruntime/test/onnx/main.cc
@@ -615,7 +615,7 @@ int real_main(int argc, char* argv[], Ort::Env& env) {
             std::string str = str_stream.str();
             ORT_THROW("Wrong value for htp_arch. select from: " + str);
           }
-        } else if (key == "enable_htp_fp16_precision" || key == "offload_graph_io_quantization" || key == "extended_udma") {
+        } else if (key == "enable_htp_fp16_precision" || key == "offload_graph_io_quantization" || key == "extended_udma" || key == "disable_htp_monolithic_lstm") {
           std::unordered_set<std::string> supported_options = {"0", "1"};
           if (supported_options.find(value) == supported_options.end()) {
             std::ostringstream str_stream;
--
2.53.0.windows.1
