From f871f0df4bf6a9ab00706db56cb2175d2cd5d297 Mon Sep 17 00:00:00 2001
From: Badri Narayanan <mbadnara@qti.qualcomm.com>
Date: Tue, 21 Jul 2026 11:40:16 -0700
Subject: [PATCH 4/4] Update argument for monolithic lstm

---
 onnxruntime/test/onnx/main.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/onnxruntime/test/onnx/main.cc b/onnxruntime/test/onnx/main.cc
index ae7597538..dcb09f927 100644
--- a/onnxruntime/test/onnx/main.cc
+++ b/onnxruntime/test/onnx/main.cc
@@ -611,7 +611,7 @@ int real_main(int argc, char* argv[], Ort::Env& env) {
             std::string str = str_stream.str();
             ORT_THROW("Wrong value for htp_arch. select from: " + str);
           }
-        } else if (key == "enable_htp_fp16_precision" || key == "offload_graph_io_quantization" || key == "extended_udma") {
+        } else if (key == "enable_htp_fp16_precision" || key == "offload_graph_io_quantization" || key == "extended_udma" || key == "enable_htp_monolithic_lstm") {
           std::unordered_set<std::string> supported_options = {"0", "1"};
           if (supported_options.find(value) == supported_options.end()) {
             std::ostringstream str_stream;
-- 
2.53.0.windows.1

